scripts/dev_setup.sh

```diff
@@ -82,53 +57,6 @@ rustup update
 rustup component add rustfmt
 rustup component add clippy
 
-if [[ $"$PACKAGE_MANAGER" == "apt-get" ]]; then
-	echo "Updating apt-get......"
-	sudo apt-get update
-fi
-
-echo "Installing CMake......"
-if which cmake &>/dev/null; then
-	echo "CMake is already installed"
-else
-	if [[ "$PACKAGE_MANAGER" == "yum" ]]; then
-		sudo yum install cmake -y
-	elif [[ "$PACKAGE_MANAGER" == "apt-get" ]]; then
-		sudo apt-get install cmake -y
-	elif [[ "$PACKAGE_MANAGER" == "pacman" ]]; then
-		sudo pacman -Syu cmake --noconfirm
-	elif [[ "$PACKAGE_MANAGER" == "brew" ]]; then
-		brew install cmake
-	fi
-fi
```

Comment:
Can we keep this block? We still need cmake, as a camper just ran into this issue on osx last week.